custom/icds_reports/static/js/angular-services/locations.service.js

```diff
@@ -1,6 +1,13 @@
 window.angular.module('icdsApp').factory('locationsService', ['$http', '$location', 'storageService', 'navigationService', function ($http, $location, storageService, navigationService) {
     var url = hqImport('hqwebapp/js/initial_page_data').reverse;
     var gtag = hqImport('analytix/js/google').track;
+    var ALL_OPTION =  {
+        name: 'All',
```

Comment:
should this be translated?

Reply:
nothing in the dashboard is translated now, so its probably not necessary